### DISCUSSION TITLE
[FEAT] Adding completed tasks at the bottom of the list

### DISCUSSION THREAD

**Discussion by SilentVoid13** [vslinko/obsidian-outliner#165] [Ideas]
## Description

I think it could be a great option (that you could toggle in the settings) to be able to automatically move completed tasks at the end of the task list.

## Example

Let's say you have the following task list:

```
- [ ] A
- [ ] B
- [ ] C
- [ ] D
- [x] E
```

Let's say you mark the `B` task as complete, it would automatically be moved to the bottom like so:

```
- [ ] A
- [ ] C
- [ ] D
- [x] B
- [x] E
```

Thanks for the great plugin!

**Comment by runchard** [comment]
Maybe even more general, a command to move current item in list to the end. 

And of course, this method may not follow the order of finished items above, which has many conditions that need to be maintained.
